packages/SwingSet/src/devices/lib/deviceTools.js

```diff
@@ -74,11 +74,11 @@ export function buildSerializationTools(syscall, deviceName) {
   }
 
   const m = makeMarshal(convertValToSlot, convertSlotToVal, {
-    marshalName: `device:${deviceName}`,
+    marshalName: `deviceTools:${deviceName}`,
```

Comment:
The prefix `device:` is also used by `deviceSlots.js`, so changing this one to be distinct.